Virustotal dbotscore duplication fix (#5021)
demisto/content: Integrations/integration-VirusTotal_CHANGELOG.md

## [Unreleased]
  - Added batch support for the **ip** and **url** and **domain** commands.

>   - Fixed an issue Where DBotScore would create duplications in context. This will take effect from Demisto version 5.5.

Reviewer: ```suggestion
  - Fixed an issue where the DBotScore would create duplications in the incident context. This effects Demisto version 5.5 and higher.
```
